content/docs/06.workflow-components/05.inputs.md

```diff
@@ -35,25 +35,38 @@ namespace: example
 inputs:
   - id: string
     type: STRING
+    defaults: "Hello World!"
 
   - id: optional
     type: STRING
     required: false
 
   - id: int
     type: INT
+    defaults: 100
 
   - id: bool
     type: BOOLEAN
+    defaults: true
 
   - id: float
     type: FLOAT
+    defaults: 100
```

Comment:
Can we change this to include a decimal point so it's different to the INT? 100.3 for example

Reply:
ohh sure.. looks to be a miss..